templates/bugscore.html

```diff
@@ -0,0 +1,302 @@
+<style>
+body {
+    font-family: Arial, sans-serif;
+    margin: 20px;
+    background-color: #f4f4f4;
+    color: #333;
+}
+
+table {
+    width: 100%;
+    border-collapse: collapse;
+    margin-top: 20px;
+}
+
+table, th, td {
+    border: 1px solid #ddd;
+}
+
+th {
+    background-color: #007bff;
+    color: white;
+    padding: 10px;
+    cursor: pointer;
+}
+
+td {
+    padding: 8px;
+    text-align: left;
+}
+
+tr:nth-child(even) {
+    background-color: #f2f2f2;
+}
+
+tr:hover {
+    background-color: #ddd;
+}
+
+input[type="number"], select {
+    padding: 5px;
+    margin: 5px 0;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+}
+
+input[type="checkbox"] {
+    margin-right: 5px;
+}
+
+.filter-section {
+    background-color: white;
+    padding: 15px;
+    border-radius: 5px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.filter-section td {
+    padding: 5px;
+}
+
+.asc:after {
+    content: " \25B2"; /* Upwards arrow */
+}
+
+.desc:after {
+    content: " \25BC"; /* Downwards arrow */
+}
+</style>
+
+<script>
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
I do not think we could use js in emails.

Reply:
nope, we can't :)
but this doesn't matter much. I will export the html on a website

Reply:
We could make the bot export the HTML page. Some of the simple options could be a GCS bucket or GitHub pages (based on the same repo or a new repo). A subdomain could point to that page (e.g., `reports.moz.tools`).

We could also do the same for other rules so that the web pages will have the latest updated data.